internal/rbd/replicationcontrollerserver_test.go

```diff
@@ -98,10 +98,10 @@ func TestValidateSchedulingDetails(t *testing.T) {
 		{
 			"when mirroring mode is journal",
 			map[string]string{
-				imageMirroringKey:     "journal",
+				imageMirroringKey:     string(imageMirrorModeJournal),
 				schedulingIntervalKey: "1h",
```

Comment:
The `schedulingIntervalKey` is not used with journal-mode? Does this need some validation and warn about a parameter that is not used?

Reply:
Sure!